website/client/src/store/actions/auth.js

```diff
@@ -108,3 +109,26 @@ export function logout (store, options = {}) {
   const query = options.redirectToLogin === true ? '?redirectToLogin=true' : '';
   window.location.href = `/logout-server${query}`;
 }
+
+export async function facebookOrGoogleAuth (store, params) {
+  const { network, redirectUrl } = params;
+  try {
+    await socialLogout(network);
+  } catch (e) {} // eslint-disable-line
+
+  try {
+    const auth = await socialLogin(network, redirectUrl);
+
+    await store.dispatch('auth:socialAuth', {
+      auth,
+    });
+
+    setUpAxios();
```

Comment:
Is there a reason why the original code (`window.location.href = '/';`) was changed to

```
setUpAxios();
await store.dispatch('user:fetch', { forceLoad: true });
```

?

Reply:
well, my bad, authForm.vue and home.vue got different behaviors after dispatch socialAuth actions